concepts/decorators/about.md

```diff
@@ -1,2 +1,210 @@
-#TODO: Add about for this concept.
+# About
 
+Decorators are a feature of the Python language that can modify or register a function.
+
+## How to use them
+
+Decorators are placed just above a function in the code like so:
+
+```py
+@decorator
+def function():
+    pass
+```
+
+Some decorators take arguments:
+
+```py
+@decorator2(name="Bob")
+def function2():
+    pass
+```
+
+If a decorator takes any arguments, but you don't wish to pass any, you must still call it with parentheses for the decorator to work:
+
+```py
+@decorator3()
+def function3():
+    pass
+```
+
+Decorators are just syntactic sugar.
+An alternative way of doing the same as the above three examples is using higher-order functions:
+```py
+def function():
+    pass
+
+function = decorator(function)
+
+
+def function2():
+    pass
+
+function2 = decorator2(name="Bob")(function2)
+
+
+def function3():
+    pass
+
+function3 = decorator3()(function3)
+```
+
+## How to write them
+
+As said before, decorators can modify or register a function.
+The simplest decorator however, does absolutely nothing.
+
+Decorators are functions which return functions.
+They take one argument - the function which they have been added to.
+They also return one thing - the function which will be run by the program.
+This is usually either the function they were added to or a slightly modified version of that function, but it does not have to be - it can be any function whatsoever.

```

Comment:
Again, I think this content is better at the top, in the first section.   Here in this section, I would suggest having a complete, simple decorator in a code example with comments to explain.  Then you can elaborate, go through the steps of writing one, and perhaps extend the original code through successive examples?   Here is an annotated starting example from the PyBit.es article:

![image](https://user-images.githubusercontent.com/5923094/148606627-b624a97b-1f11-4c86-962a-68c2925c6636.png)